wiki/en/Privacy-Statement.md

```diff
@@ -7,41 +7,43 @@ permalink: "/wiki/Privacy-Statement"
+ 
 
 ## Definition of Terms
 
-- "**Server**" The Jamulus Server software, as opposed to the host machine/OS it is running on.
-- "**Client**" The Jamulus software used to connect to a Server
+- "**Server**" The Jamulus Server software running on a remote computer to which you will connect.
+- "**Client**" The Jamulus software running on your local computer used to connect to a Server
 - "**Directory**" A Jamulus Server configured to supply a list of Servers to Clients
 
-## Web site
+## Jamulus.io Web site
 
-The website at [jamulus.io](https://jamulus.io) is served using GitHub Pages. We do not collect your personal data or set tracking cookies. When you connect to the jamulus.io site, your IP is sent to [GitHub Pages](https://pages.github.com/).
+The website at [jamulus.io](https://jamulus.io) is served using GitHub Pages. See Github's [privacy policy](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement) for information relating to data collection and privacy.
 
 ## Jamulus Software
 
-### Use of Profile Information
+Jamulus is open source software and can therefore be modified by others.  As such, the Jamulus project makes no representations related to privacy, data collection, or security with respect to your use of the software.  
 
-When you connect to a Server, whatever you put in My Profile (in Settings) will be shown to others on that Server while you are connected to it.  Note that, depending on the other client, you may not see them.  Also, the Server operator can see the name you set in My Profile and your IP address, using the Server either through the GUI or JSON-RPC.
+## General Information
 
-When you connect to a Server, your profile may also be available to third parties from the Directory to which that Server is registered. This can be for informational purposes about the status of the public Jamulus network (for example, [here](https://explorer.jamulus.io/)), but may not be limited to that. Profile information is not otherwise logged or stored by the Jamulus Server you are connected to, or by the Jamulus Directory, but may be stored or processed by third parties.
+Under normal use with unmodified software, your user profile information is exchanged with Servers you connect to, peers connected to those same Servers, and to any interested third party (including Directories) that uses the Jamulus protocol.  This information is limited to your Jamulus name, city, country, instrument, and skill level as you have set them in your profile.  The Servers you connect to will also have access to your internet address (IP Address) as it is required for the software to work; but this information is not shared with peers on the same Server or normally available to third parties. 
```

Comment:
```suggestion
Under normal use with unmodified software, your user profile information is exchanged with the Servers you connect to, peers connected to those same Servers, and to any interested third party (including Directories) that uses the Jamulus protocol. This information is limited to your Jamulus name, city, country, instrument, and skill level as you have set them in your profile. The Servers you connect to will also have access to your internet address (IP Address) as it is required for the software to work; however, this information is not shared with peers on the same Server nor is it normally available to third parties.
```

Reply:
Not sure if I understood that last sentence correctly. Anyway, "normally" is a bit ambiguous isn't it? Under what circumstances **is** it available to third parties?

Reply:
Fair point, I was reinforcing that a "modified" server may share that.  I think I can remove that final qualification.

Reply:
Since we can't even begin to guess or cover what a "modified" version might do, I think it should be clearly stated somewhere that this privacy statement only applies to the Jamulus version downloaded through our website (like the previous statement did).